src/coreComponents/fileIO/vtk/VTKPVDWriter.cpp

```diff
@@ -42,20 +42,20 @@ void VTKPVDWriter::setFileName( string fileName )
 
 void VTKPVDWriter::save() const
 {
-  m_pvdFile.save_file( m_fileName.c_str() );
+  m_pvdFile.saveFile( m_fileName.c_str() );
 }
 
 void VTKPVDWriter::addData( real64 time, string const & filePath ) const
 {
-  auto collectionNode = m_pvdFile.child( "VTKFile" ).child( "Collection" );
+  auto collectionNode = m_pvdFile.getChild( "VTKFile" ).child( "Collection" );
```

Comment:
I can wrap `xml_node` and `xml_attribute` the same way I wrapped `xml_document`. It also implies to change all `attribute()` to `getAttribute()`, all `name` and `value` to `getName` and `getValue`.
It involves quite a few changes, but nothing impossible. It would uniformize this part of the code and it would be a "true" wrapper (more than a list of `using`).

I think I should use inlines here, or do you think I should continue with separated method in the cpp?

Reply:
Let's to another PR for this, I think it's best.
w.r.t. the inline policy, if it's onliners, surely `inline` is nice, but I'm fine with your decisions.